ahiv/kafka/protocol/packet/metadata.h

```diff
@@ -40,7 +40,7 @@ struct MetadataRequestPacket : public RequestPacket {
     buffer.Overwrite<int32_t>(packetSizePosition, packetSize);
   }
 
-  std::size_t Size() {
+  std::size_t Size() override {
     std::size_t packetSize = RequestPacket::Size() + 4 + 3;
```

Comment:
This again seems like a magic number to me. Even though the writer knows that 3 and 4 are the sizes of the written fields, the reader may be confused.

Reply:
What should be the solution for this? Write constants for all types and comment document string lengths?

Reply:
that would be great